### DISCUSSION TITLE
Pot de miel et accessibilité

### DISCUSSION THREAD

**Discussion by econuweb** [cnumr/best-practices-wordpress#184] [Bonnes pratiques]
BP : https://github.com/cnumr/best-practices-wordpress/blob/main/fiches/WP_1.04-securiser-les-formulaires-en-ligne.md

À propos de la solution "pot de miel", il me semble important de préciser que le champ piège ne doit pas être accessible aux utilisateurs qui naviguent avec le clavier (ou au moins qu'ils soient dissuadés de le remplir) ; question d'accessibilité.

**Comment by dNicolle** [comment]
> BP : https://github.com/cnumr/best-practices-wordpress/blob/main/fiches/WP_1.04-securiser-les-formulaires-en-ligne.md
> 
> À propos de la solution "pot de miel", il me semble important de préciser que le champ piège ne doit pas être accessible aux utilisateurs qui naviguent avec le clavier (ou au moins qu'ils soient dissuadés de le remplir) ; question d'accessibilité.

Très bonne remarque merci.
J'y ajouterais bien le code d'un honey pot accessible non ? ou un lien vers une source valable ?

**Comment by EmmanuelDemey** [comment]
Il faudrait donc ajouter un tabindex=-1?
